src/shared_modules/utils/stringHelper.h

```diff
@@ -310,7 +305,43 @@ namespace Utils
 
         return !str.empty() && it == str.end();
     }
-}
+
+    static int parseStrToBool(const std::string& str)
+    {
+        return (str.compare("yes") == 0) ? 1 : (str.compare("no") == 0) ? 0 : -1;
+    }
+
+    static long parseStrToTime(const std::string str)
```

Comment:
Please, add tests for this helper.
There are basic cases that aren't working

https://godbolt.org/z/j7jaeb4xo

Reply:
Done thx guys!